specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2018-02-01-preview/containerInstance.json

```diff
@@ -0,0 +1,1099 @@
+        },
+        "storageAccountName": {
+          "description": "The name of the storage account that contains the Azure File share.",
+          "type": "string"
+        },
+        "storageAccountKey": {
+          "description": "The storage account access key used to access the Azure File share.",
+          "type": "string"
+        }
+      }
+    },
+    "EmptyDirVolume": {
+      "description": "The empty directory volume.",
+      "type": "object",
+      "properties": {}
+    },
+    "SecretVolume": {
+      "description": "The secret volume.",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "GitRepoVolume": {
+      "description": "Represents a volume that is populated with the contents of a git repository",
+      "required": [
+       "repository"
+      ],
+      "properties": {
+        "directory": {
+          "description": "Target directory name. Must not contain or start with '..'.  If '.' is supplied, the volume directory will be the git repository.  Otherwise, if specified, the volume will contain the git repository in the subdirectory with the given name.",
+          "type": "string"
+        },
+        "repository": {
+          "description": "Repository URL",
+          "type": "string"
+        },
+        "revision": {
+          "description": "Commit hash for the specified revision.",
+          "type": "string"
+        }
+      }
+    },
+    "Volume": {
+      "description": "The properties of the volume.",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the volume.",
+          "type": "string"
+        },
+        "azureFile": {
+          "description": "The Azure File volume.",
+          "$ref": "#/definitions/AzureFileVolume"
+        },
+        "emptyDir": {
+          "description": "The empty directory volume.",
+          "$ref": "#/definitions/EmptyDirVolume"
+        },
+        "secret": {
+          "description": "The secret volume.",
+          "$ref": "#/definitions/SecretVolume"
+        },
+        "gitRepo": {
+          "description": "The git repo volume.",
+          "$ref": "#/definitions/GitRepoVolume"
+        }
+      }
+    },
+    "VolumeMount": {
+      "description": "The properties of the volume mount.",
+      "type": "object",
+      "required": [
+        "name",
+        "mountPath"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the volume mount.",
+          "type": "string"
+        },
+        "mountPath": {
+          "description": "The path within the container where the volume should be mounted. Must not contain colon (:).",
+          "type": "string"
+        },
+        "readOnly": {
+          "description": "The flag indicating whether the volume mount is read-only.",
+          "type": "boolean"
+        }
+      }
+    },
+    "ContainerGroup": {
```

Comment:
Tracked resource 'ContainerGroup' must have patch operation that at least supports the update of tags. It's strongly recommended that the PATCH operation supports update of all mutable properties as well.

Reply:
There is another PR for patch operation. I combined the changes from that PR to this one.